static/src/javascripts/bootstraps/standalone.commercial.ts

```diff
@@ -170,7 +171,18 @@ const bootCommercial = async (): Promise<void> => {
 		const promises = allModules.map((args) => loadModules(...args));
 
 		await Promise.all(promises).then(() => {
-			EventTimer.get().trigger('commercialModulesLoaded');
+			const eventTimer = EventTimer.get();
```

Comment:
It may be worth extracting the slot counting logic into its own function / module and simply call that here?